apps/hello/main.py

```diff
@@ -5,10 +5,10 @@
 import statistics
 
 from streamsync.core import StreamsyncState
+from streamsync.ui import StreamsyncUIManager
```

Comment:
Are we using CMCs?

Reply:
I was using it at first but removed it in the end.